filesystem/File.php

```diff
@@ -466,35 +432,12 @@ public static function get_app_category($ext) {
 	/**
 	 * Returns a category based on the file extension.
 	 *
-	 * @return String
+	 * @return string
 	 */
 	public function appCategory() {
 		return self::get_app_category($this->getExtension());
 	}
 
-	public function CMSThumbnail() {
-		return '<img src="' . $this->Icon() . '" />';
-	}
-
-	/**
-	 * Return the relative URL of an icon for the file type,
-	 * based on the {@link appCategory()} value.
-	 * Images are searched for in "framework/images/app_icons/".
-	 *
-	 * @return String
-	 */
-	public function Icon() {
```

Comment:
Is this restored somewhere else? What's the thinking behind just removing this, but deprecated Image? Just - it's hard to deprecate?